src/routes/extras/containers/ExtrasPage.js

```diff
@@ -60,19 +42,21 @@ export class ExtrasPage extends React.Component {
   }
 
   getContent () {
-    const {t} = this.props
-    const sprungbrett = this.props.extras.find(extra => extra.alias === SPRUNGBRETT_EXTRA)
+    const LoadingSpinner = () => <Spinner name='line-scale-party' />
 
-    if (this.props.extra === SPRUNGBRETT_EXTRA && sprungbrett) {
-      return <SprungbrettPage title={sprungbrett.name} />
-    } else if (this.props.extra) {
+    const {extraAlias, extras, sprungbrettJobs} = this.props
+
+    if (extraAlias) {
+      const extra = extras.find(_extra => _extra.alias === extraAlias)
+
+      if (extra && extraAlias === SPRUNGBRETT_EXTRA) {
+        return sprungbrettJobs ? <SprungbrettList title={extra.name} jobs={sprungbrettJobs} /> : <LoadingSpinner />
```

Comment:
What happens if there occurs an error while fetching? 🤔

Reply:
error handling is still missing

Reply:
I would do this whole error handling topic in WEBAPP-203, this task is already large enough imo